NavigationSystem/makefile

```diff
@@ -29,8 +33,7 @@ TOOLCHAIN = 0
 
 
 export CPPFLAGS                = -g -Wall -pedantic -Werror -std=gnu++14
-
-export LIBS                    = -lsqlite3 -lgps -lrt -lcurl -lpthread -lwiringPi -lncurses
+export LIBS                    = -lsqlite3 -lgps -lrt -lcurl -lpthread -lwiringPi
```

Comment:
I think that the -lncurses link needs to be added. Need to check. It might be used in the voter system. Not sure at all